src/HPWH.cc

```diff
@@ -732,11 +648,8 @@ int HPWH::runOneStep(double drawVolume_L,
             if (heatSources[i].isLockedOut() && heatSources[i].backupHeatSource == NULL)
             {
                 heatSources[i].disengageHeatSource();
-                if (hpwhVerbosity >= HPWH::VRB_emetic)
-                {
-                    msg("\nWARNING: lock-out triggered, but no backupHeatSource defined. "
-                        "Simulation will continue will lock out the heat source.");
-                }
+                send_warning("lock-out triggered, but no backupHeatSource defined. Simulation will "
+                             "continue will lock out the heat source.");
```

Comment:
Maybe clean up the message while we're at it? It doesn't read well, and I think there might be a typo in it.